### ISSUE TITLE
Batch normalization folding

### ISSUE THREAD

**Issue by kparichay** [nnstreamer/nntrainer#869]
Once the model is trained, batch normalization is fixed linear operations, and it can be fused into the previous layer.
This is called folding the batch normalization layer - https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/graph_transforms/README.md#fold_batch_norms.

This must be supported with nntrainer, for inferring the model with lower latency after training.
